src/test/unit/wrappers/http/test_sync_http_wrappers.py

```diff
@@ -151,13 +151,18 @@ def lambda_test_function(event, context):
 
 def test_catch_file_like_object_sent_on_http(context, token):
     class A:
+        done = False
```

Comment:
This is a fix to a flaky test (work in some machines, and doesn't work in other)